analysis_options.yaml

```diff
@@ -17,4 +17,4 @@ linter:
 
     # TODO: the following rules were temporarily disabled to make it easier to
     # transition to flutter_lints 2.0 that has these new rules enabled by default
-    depend_on_referenced_packages: false
+  
```

Comment:
we could remove the comment and the empty lines above since this is the last remaining temporarily disabled lint rule

Reply:
can you be more specific ? what do to ?

Reply:
✅

Reply:
please remove the whole lines without leaving blank spaces :)

![image](https://user-images.githubusercontent.com/140617/171587550-57e9d163-687c-49df-be4d-f5ac18d8738a.png)

Reply:
✅